ci/container/Dockerfile

```diff
@@ -55,6 +55,25 @@ RUN export DEBIAN_FRONTEND=noninteractive && \
     cd .. && \
     rm -rf e2fsdroid
 
+# Install an older libtinfo for rules_haskell
+RUN mkdir libtinfo && \
```

Comment:
would this work too?
```
apt install libtinfo6
```

and if not, can you add a note about why we install it from the deb? is v5 necessary?

Reply:
Looks like libtinfo5 is needed indeed: https://github.com/tweag/rules_haskell/blob/master/docs/haskell.rst#before-you-begin

Disregard my comment; I'll try to see why rules_haskell needs it though